src/rewrite_app_repos.sh

```diff
@@ -52,13 +52,11 @@ do
   do
     if [ -e "$f" ]; then
       json_path=".require.\"greenpeace/${reponame}\""
-      plugin_version=$(jq -r "${json_path}" < "${f}")
+      plugin_version=$(jq -r "${json_path} // empty" < "${f}")
       branch=${plugin_version#"$composer_dev_prefix"}
 
-      # if these are the same do nothing as plugin is not in file or it doesn't start with "dev-"
-      # master already includes built files
-      if [[ "${branch}" != "${plugin_version}" ]] && [ "${branch}" != master ]; then
-        # dev branches do not include the built assets, only master does. So we need to build them here.
+      if [ -n "$branch" ]; then
```

Comment:
It probably also needs to build if the plugin is required at a specific version instead of through `master` branch?

Reply:
`$branch` actually is set with a tag also, when a tag is used in a composer file. So it has the proper value, without a `dev-` prefix when a tag is used.

This also works with `git clone`, that's a few lines below. eg.
```
git clone --recurse-submodules --single-branch --branch v1.78.2 https://github.com/greenpeace/planet4-master-theme
```

Reply:
Here is an example:
https://circleci.com/gh/greenpeace/planet4-test-tavros/55

Reply:
Ah yeah, I missed that `"${branch}" != "${plugin_version}"` was the check that was ensuring that only `dev-` versions are built, but that has been removed, so you can ignore my comment.